<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataPatternWidget</class>
 <widget class="QWidget" name="DataPatternWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>670</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="mplwindow">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6000</width>
       <height>6000</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="mplvl">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="info_groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Info</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextBrowser" name="infotext">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;Total counts: 0; Valid: 0&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;Active pixels: 0; Masked: 0&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;Angular range (x, y): 0, 0&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;Pattern orientation (x, y, phi): 0, 0, 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="acceptRichText">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mask_groupBox">
        <property name="title">
         <string>Mask</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="pb_maskpixel">
           <property name="text">
            <string>Mask Individual Pixels</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pb_maskrectangle">
           <property name="text">
            <string>Mask Rectangle</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pb_removecentral">
           <property name="text">
            <string>Mask Quad Pixels</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pb_maskedge">
           <property name="text">
            <string>Mask Edge Pixels</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="pb_savemask">
           <property name="text">
            <string>Save Mask</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="pb_loadmask">
           <property name="text">
            <string>Load Mask</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pb_maskbelow">
           <property name="text">
            <string>Mask Below</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pb_maskabove">
           <property name="text">
            <string>Mask Above</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pb_expandmask">
           <property name="text">
            <string>Expand Mask</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="pb_clearmask">
           <property name="text">
            <string>Clear Mask</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_mask_group</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Pattern Manipulation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1" colspan="2">
          <widget class="QPushButton" name="pb_buildmesh">
           <property name="text">
            <string>Build Angular Calibration Mesh</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_manip_group</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QPushButton" name="pb_fitrange">
           <property name="text">
            <string>Set Fit Range</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_manip_group</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pb_orientchanneling">
           <property name="text">
            <string>Orient Channeling Axis</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_manip_group</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pb_editorientation">
           <property name="text">
            <string>Edit Orientation</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_manip_group</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QPushButton" name="pb_compressmesh">
           <property name="text">
            <string>Compress Pixel Mesh</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_manip_group</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="pb_setticks">
        <property name="title">
         <string>Pattern Visualization</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pb_colorscale">
           <property name="text">
            <string>Edit Color Scale</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_vis_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_setlabels">
           <property name="text">
            <string>Set Labels</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_vis_group</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_vis_group">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="bg_manip_group">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="bg_mask_group">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
